src/pickup.cpp

```diff
@@ -169,6 +182,49 @@ interact_results interact_with_vehicle( vehicle *veh, const tripoint &pos,
             veh_tool( "hotplate" );
             return DONE;
 
+        case USE_WASHMACHINE: {
+            bool detergent_is_enough = g->u.crafting_inventory().has_charges( "detergent", 5 );
+            static const std::string filthy( "FILTHY" );
+            bool filthy_items = std::any_of( veh->get_items( washing_machine_part ).begin(),
+                                             veh->get_items( washing_machine_part ).end(),
```

Comment:
Implemented in 6114c40, but had to made it without `const`, as `items` are changed below, where we set the `bday`.